samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake_tests/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dev_dependencies:
   built_collection: 5.1.1
   built_value: 8.4.3
-  dio: 5.2.1
+  dio: 5.7.0
```

Comment:
Can you please change this to `dio: '^5.7.0'` like in the sibling folders?

Reply:
Yea please add it, otherwise we will miss this again in the future.

Reply:
update via https://github.com/OpenAPITools/openapi-generator/pull/20797/commits/4ced1d392468e3cfa3b90691e0d2fe86789a6b25